openstack/nova/templates/consoleauth-deployment.yaml

```diff
@@ -74,6 +75,7 @@ spec:
           mountPath: /etc/nova/logging.ini
           subPath: logging.ini
           readOnly: true
+{{- end }}
 {{ $envAll := . }}
 {{- range $name, $config := .Values.consoles }}
   {{- if $config.enabled }}
```

Comment:
Do you plan to remove this part when we finally remove nova-consoleauth from the helm-chart?

Reply:
Line 1-78? Yes, that should go. Anything between `if .Values.pod.replicas.consoleauth` and `end`